en/docs/develop-components/manage-component-source-configurations.md

```diff
@@ -39,8 +39,7 @@ endpoints:
     displayName: Go Greeter Sample
     # +required Service section has the user service endpoint details
     service:
-      # +optional Base path of the API that gets exposed via the endpoint.
-      # This is mandatory if the endpoint type is set to REST or GraphQL.
+      # +optional Base path of the endpoint.
```

Comment:
Here we expect the basePath of the backend service (API), so having `Base path of the endpoint` can be confusing.

```suggestion
      # +optional  Context (base path) of the API that gets exposed via the endpoint.
```

Reply:
Addressed via https://github.com/wso2/docs-choreo-dev/pull/1628